.yarnrc.yml

```diff
@@ -22,4 +22,4 @@ yarnPath: .yarn/releases/yarn-3.4.1.cjs
 packageExtensions:
   '@wdio/browser-runner@*':
     dependencies:
-      '@babel/core': '*'
+      '@babel/core': 'patch:@babel/core@npm%3A7.21.0#./.yarn/patches/@babel-core-npm-7.21.0-fb3817b0e5.patch::locator=root%40workspace%3A'
```

Comment:
?

Reply:
I added some more stuff to `packageExtensions` and replaced the Babel version with the one we use in `package.json`, but later ended up removing those additions to `packageExtensions`. 😅